tests/frontend/unit/StatusBadge.spec.js

```diff
@@ -0,0 +1,16 @@
+import StatusBadge from '@DpJs/components/procedure/Shared/StatusBadge'
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+describe('StatusBadge.vue', () => {
+  it('renders correctly for each status', () => {
```

Comment:
Not a blocker: I think it is better to create an 'it' statement for each test case. That way, if it fails, you know instantly what is broken. If all test cases are in a single test, and the test fails, any of the test cases may be broken.